### PR TITLE
Add goerli testnet

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

**Comment by mandrigin** [comment]
@dshulyak thanks! we probably need a status-go update too, right?

**Comment by dshulyak** [comment]
@mandrigin yes, i had some problems with a status-go build. i will make a status-go release a bit later

File: STATUS_GO_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.23.0-beta.8
+190315-072737-3049ad
```
Comment (mandrigin) [inline STATUS_GO_VERSION]: I think this change is unnecessary, otherwise looks good.

**Review by mandrigin — CHANGES_REQUESTED** [review]

**Comment by mandrigin** [comment]
@dshulyak can you maybe update the status-go version to the current one, then we can merge it?

**Comment by pedropombeiro** [comment]
By the way, just to make sure we're on the same page, it's no longer necessary to build status-go on Jenkins before building with updated version in status-react, since Nix will build status-go package as part of its build.

**Comment by flexsurfer** [comment]
@mandrigin @dshulyak is this still relevant?

**Comment by mandrigin** [comment]
we probably should rebase, test and merge it

**Comment by jakubgs** [comment]
I'd like to revive this. Ropsten has been getting worse and worse in terms of syncing:
rops_les_sync
https://grafana.status.im/d/VyUgFRWZz/geth-syncing?orgId=1&var-fleet=rops.les&from=now-2d&to=now
faucet_ropsten_sync
https://grafana.status.im/d/VyUgFRWZz/geth-syncing?orgId=1&from=now-13d&to=now&var-fleet=faucet.ropsten
I'd love to switch to the [Görli](https://goerli.net/)(`goerli`) network so we could replace `les.rops` and `faucet.ropsten` fleets with `les.goerli` and `faucet.goerli`. I intend to deploy these two today and see how well it sync.

**Comment by jakubgs** [comment]
Created a Discuss post about this:
https://discuss.status.im/t/replacing-ropsten-with-gorli-network-for-faucet-and-les-needs/1235

**Review by mandrigin — APPROVED** [review]

**Comment by mandrigin** [comment]
@flexsurfer ready to be reviewed, tested an merged ;-)

**Review by yenda — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by asemiankevich** [comment]
@dshulyak @mandrigin do we have a faucet here? Wondering if i can test transactions

**Comment by mandrigin** [comment]
@asemiankevich not yet, but @jakubgs wanted to try moving our fauced from ropsten to goerli.

@jakubgs do we have some test faucet for it?

**Comment by asemiankevich** [comment]
What i have noticed so far:  
- using status-im.github.io/dapp I cant sign Typed message ( transaction -> sign Typed message). wrong password appears. @flexsurfer maybe you could clarify here?
- none of transactions are mined, 0 confirmations  :( Seems to be expected as not many users are using this network?

**Comment by mandrigin** [comment]
@asemiankevich hmm, that is interesting... I wonder how can we test goerli separately from Status?

**Comment by asemiankevich** [comment]
> @asemiankevich hmm, that is interesting... I wonder how can we test goerli separately from Status?

metamask should work, i can try there

**Comment by asemiankevich** [comment]
@mandrigin this Sign typed message does not work on Goerli in metamask at all, the call is not being performed, i dont see any sign message screen. Also i found that transaction history is not populated 
image

@mandrigin we could probably merge the PR, since i can switch to this network and send some messages and transactions, issues with history, mining and sign typed message i believe are network-related and we can work on this later, right? cc @annadanchenko

**Comment by jakubgs** [comment]
>@jakubgs do we have some test faucet for it?

@mandrigin @asemiankevich We do now: https://faucet-goerli.status.im/faucet-info

**Pull request merged**